README.md

```diff
@@ -66,9 +66,33 @@ class Person
 end
 ```
 
+####Attribute Accessors
+If we know that we are going to be reading and writing to the `name` attribute on our `Person` class, we can use another macro called an attribute accessor.
+
+So this code:
+
+```ruby
+class Person
+  attr_writer :name
+  attr_reader :name
+
+end
+```
+
+Can now be written like this:
+
+```ruby
+class Person
+  attr_accessor :name
+
+end
+```
+We now have access to reader and writer methods from one macro!
+
 #### Using Macros is Usually Better
 
-The usage of macros is preferred over the explicit definition of setter and getter methods, *unless you need to customize the implementation of a method*, like in our previous lesson when we defined `.name` as returning the first and last name variables combined. 
+The usage of macros is preferred over the explicit definition of setter and getter methods, *unless you need to customize the implementation of a method*, like in our previous lesson when we defined `.name` as returning the first and last name variables combined.
 
 To understand why, simply look at the code above. Which class would you rather write? The longer one, or the shorter one?
 
+When opening up a class, it is hard to realize that all of those setters/getters are kinda fluff. You don't want to have to sift through all the kinda BS-y methods to try and find the goodness. `attr_accessor` and friends allow you to get a high level overview of the class at the very beginning.
```

Comment:
I think this is a stream of conciousness thing I wrote somewhere. Can you formalize and make it not stick out so much. Probably shouldn't use things like kinda and BS-y.

Another thing to add, is that as developers we spend more time reading rather then writing code, so it's important that the code that we are reading is the hard stuff, not the stuff that's boring